src/qt/sendcoinsdialog.cpp

```diff
@@ -75,24 +76,27 @@ SendCoinsDialog::SendCoinsDialog(const PlatformStyle *_platformStyle, QWidget *p
     if (!settings.contains("bUseInstantX"))
         settings.setValue("bUseInstantX", false);
 
-    bool fUsePrivateSend = settings.value("bUseDarkSend").toBool();
-    bool fUseInstantSend = settings.value("bUseInstantX").toBool();
-    if(fLiteMode) {
+    if (!privateSendClient.fEnablePrivateSend) {
         ui->checkUsePrivateSend->setChecked(false);
         ui->checkUsePrivateSend->setVisible(false);
-        ui->checkUseInstantSend->setVisible(false);
         CoinControlDialog::coinControl->fUsePrivateSend = false;
-        CoinControlDialog::coinControl->fUseInstantSend = false;
-    }
-    else{
+    } else {
+        bool fUsePrivateSend = settings.value("bUseDarkSend").toBool();
```

Comment:
should maybe be "PrivateSend"ivized

Reply:
Maybe but it's out of scope for this PR imo.